src/routes/test-contentful-content/+layout.svelte

```diff
@@ -0,0 +1,3 @@
+<div class="grid-container">
```

Comment:
Talked about this in Slack, but adding the `usa-prose` class here fixes some of the styling to match USWDS (heading font, tables, etc.):
```suggestion
<div class="usa-prose grid-container">
```